### PR TITLE
Spawnable pass to

### PR DESCRIPTION
**Pull request by James-Sangalli**
for #40 

Correction of misunderstanding (continue to use index model just include custom passTo), replaces #42

### CODE REVIEW

File: TicketingContract.sol [hunk 1]
```diff
@@ -224,12 +300,12 @@ contract TicketPro
...
             uint index = uint(ticketIndices[i]);
-            assert(inventory[_from][index] != bytes32(0));
+            assert(inventory[_from][index] != uint256(0));
             //pushes each element with ordering
             inventory[_to].push(inventory[msg.sender][index]);
             delete inventory[_from][index];
         }
-        
+
         emit TransferFrom(_from, _to, ticketIndices);
```
Comment (JamesSmartCell) [inline TicketingContract.sol]: Need to change the array in emitter to a count or the first ticketID. Arrays seem to produce garbage when you interpret the received return.

File: TicketingContract.sol [hunk 2]
```diff
@@ -238,9 +314,9 @@ contract TicketPro
         selfdestruct(organiser);
     }
 
-    function isStormBirdContract() public pure returns (bool) 
+    function isStormBirdContract() public pure returns (bool)
     {
-        return true; 
+        return true;
     }
```
Comment (JamesSmartCell) [inline TicketingContract.sol]: Need to update this to return which interfaces are available so the client can interact with the contract correctly.

**Comment by colourful-land** [comment]
Hi you missed this spot from the previous comment:

"Also, these functions take a parameter of TokenID in byte32 according to the XML but we don't really need to care about the vaue used in that parameter - we can uniformly return the same street, buildingName and state value according to the constructor disregarding tokenID, for now."

**Comment by colourful-land** [comment]
1) make a new xml file. leave the existing one. 2) make sure the contract address referenced in the XML file exists. You might want get #39 approved before this.

**Comment by colourful-land** [comment]
@James-Sangalli can you have a look if @JamesSmartCell 's concerns are addressed. then this can go.

**Pull request merged**